lib/pronto/config_file.rb

```diff
@@ -1,6 +1,7 @@
 module Pronto
   class ConfigFile
     DEFAULT_MESSAGE_FORMAT = '%{msg}'.freeze
+    DEFAULT_WARNINGS_PER_REVIEW = 30
```

Comment:
@dimitrovv why is the default value 30? What's the GH rate limit?

Reply:
@mmozuras we are using **Github Enterprise** and with disabled rate limit, but if I try to post about 40 comments per review **GHE** returns **502**. I don't this is smth with the rate limit here, have not tried with Public Github. I suspect the server is not able to serve this request somehow. 

Thus I can disable the default value here, so the comments will be separated into different reviews only if a default value is specified via config or env var. What do you think?